src/main/java/com/cinema/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package com.cinema.controllers;
+
+import com.cinema.model.dto.UserRequestDto;
+import com.cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema")
```

Comment:
you may or may not have some additional logic in future. But for now it's redundant